src/Raven.Client/Documents/Session/ISessionDocumentTimeSeries.cs

```diff
@@ -88,7 +88,7 @@ public interface ISessionDocumentRollupTypedTimeSeries<TValues> :
     public interface ISessionDocumentTypedIncrementalTimeSeries<TValues> :
         ITimeSeriesStreamingBase<TimeSeriesEntry<TValues>>,
         ISessionDocumentDeleteTimeSeriesBase,
-        ISessionDocumentIncrementTimeSeriesBase
```

Comment:
This isn't breaking any backward compatibility? What will happen if someone was using one of the methods from that interaface?

Reply:
Yes, It breaks the behavior that was in the previous versions, but this was wrong from the first place
It should be like ```ISessionDocumentTypedTimeSeries```:
```
    public interface ISessionDocumentTypedTimeSeries<TValues> : 
        ISessionDocumentTypedAppendTimeSeriesBase<TValues>, 
        ITimeSeriesStreamingBase<TimeSeriesEntry<TValues>>, 
        ISessionDocumentDeleteTimeSeriesBase
        where TValues : new()
```

Reply:
In regular timeseries (not incremental)
You cant do something like this:
```
session.TimeSeriesFor<TestObj>("users/ayende", "Heartrate").Append(timestamp, new double[] {1.1, 2.2, 3.3});
```
you have to do it like that:
```
session.TimeSeriesFor<TestObj>("users/ayende", "Heartrate").Append(timestamp, new TestObj{
   A = 1.1,
   B = 2.2,
   C = 3.3
});
```
.
.
.
That's why I removed the ability to do that
```
session.IncrementalTimeSeriesFor<TestObj>("users/ayende", "INC:Heartrate").Increment(timestamp, new double[] {1.1, 2.2, 3.3});
```
And added the ability to do that instead:
```
session.IncrementalTimeSeriesFor<TestObj>("users/ayende", "INC:Heartrate").Increment(timestamp, new TestObj{
   A = 1.1,
   B = 2.2,
   C = 3.3
});
```

Reply:
better to stay with that in 5.4 and remove that in 6.0, we od not want to break any compatibility

Reply:
Done

Reply:
For changing it in v6.0:
https://issues.hibernatingrhinos.com/issue/RavenDB-19511/Remove-non-typed-inheritance-from-typed-incremental-time-series-client-API